app/controllers/APIController.scala

```diff
@@ -172,6 +174,26 @@ class APIController @Inject()(etmpService: ETMPService,
             Status(res.status)(Json.toJson(res.body))
           case _ =>
             logger.warn(s"[APIController][getFinancialDetails] status ${res.status} returned from EIS " +
+                s"Status code:'${res.status}', Body: '${res.body}")
```

Comment:
Is this still needed? Do we want to log the body? There could be sensitive data

Reply:
Ok. So can we keep the status code and remove the body from the logs.

Reply:
I think this should still be removed, we already have the status logged on line 176.